modin/pandas/groupby.py

```diff
@@ -19,7 +19,7 @@
 
 from modin.error_message import ErrorMessage
 
-from .utils import _inherit_docstrings, wrap_udf_function
+from .utils import _inherit_docstrings, wrap_udf_function, try_cast_to_pandas
```

Comment:
it seems that you forgot to remove import that you've used for the experiments
(at least I don't see where it uses below)